### ISSUE TITLE
Not an issue - activity feeds + opensearch

### ISSUE THREAD

**Issue by johndpope** [snarfed/bridgy#1543]
Hi @snarfed  - 

I'm playing around activity feeds - and go so to fire up opensearch on aws 
I'm tracking an issue here - I know you have the granary app - 

https://github.com/qlands/elasticfeeds/issues/

it would seem to make sense if either of these could bubble up the sorts of data / content your capturing into an activity feed  with python.

maybe not.... did you ever consider it?

more background

https://opensocial.github.io/spec/2.5.1/Social-Data.xml

<img width="1422" alt="Screenshot 2023-08-26 at 7 39 33 pm" src="https://github.com/snarfed/bridgy/assets/289994/26ed2df7-9a5c-4f50-9ae1-334e5ec022a5">

**Comment by snarfed** [comment]
Hmm! I don't know this https://www.getopensocial.com/ startup, but I know [the original OpenSocial standard itself](https://en.wikipedia.org/wiki/OpenSocial) well! Blast from the past, I knew some of the creators well during its development. And you're right to mention https://granary.io/ , [its REST API](https://github.com/snarfed/granary/#using-the-rest-api) implememts OpenSocial's Activity Streams API.

Sadly the OpenSocial standard itself is dead. I do run a few services that convert social network timelines/feeds to Atom, eg https://bluesky-atom.appspot.com/ , and https://granary.io/ can convert more, eg Mastodon. I used to run similar services for Twitter, Facebook, and Instagram, but they all either closed their APIs or otherwise became too developer hostile to keep those services working.

**Comment by johndpope** [comment]
I'm about to run off to the park with kids - but thanks coming back to me. 
Once upon a time - I used this - and it worked well. it was mysql backend. Some dude did their phd on it in spain.
https://github.com/ging/social_stream

I was looking around for modern day alternatives - I did come across this - go-lang
https://github.com/superseriousbusiness/gotosocial

I stumbed elasticfeeds as I'm using python these days.
not sure you saw the above ticket - but I managed to upgrade elasticfeeds to use opensearch - 
is this the best direction for an opensource + activityfeeds????  it seems quite powerful.
(sidenote - elasticsearch changed its licesne - but amazon took it over from fork )

I know there's this one aswell - stream_framework
https://github.com/tschellenbach/Stream-Framework/blob/master/stream_framework/feeds/aggregated_feed/notification_feed.py

I asked chatgpt for help upgrading rails / social_stream project - was some what successful - but gave up after a bit.
I do have this in a virtualbox container somewhere. hmmm... kids screaming....goto run.
